src/main/msp/msp.c

```diff
@@ -1247,7 +1247,11 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
 #else
         sbufWriteU8(dst, 0);
 #endif
-
+#if defined(USE_RC_SMOOTHING_FILTER)
```

Comment:
Please add a comment indicating the API version this was added.

Reply:
Added the comment.